lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -911,6 +911,7 @@ export class NativeInteractionClient extends BaseInteractionClient {
                 ...request.tokenQueryParameters,
             },
             extendedExpiryToken: false, // Make this configurable?
+            reqCnf: "",
```

Comment:
Why is this empty string? Should we not respect what was passed in?

Reply:
Looks like we set it to an empty string on purpose. Then for cases when auth schema is set to PoP we add `request.reqCnf` to `validatedRequest` or generate our own, if not provided in the request.
What i can't understand though is if we really need to set `reqCnf` to an empty string for non-PoP requests then why do we apply this logic only to the native flow.